Update AlertDialog import to use SupportLibrary.

yihanseattle opened the pull request:
## Motivation

Fix the button not fully shown issue on Alert. This issue happens on an Emulator with API 21, Samsung S5 with API 21 and Samsung Note 3 with API 21. 

## Before:
![screenshot_1491263293](https://cloud.githubusercontent.com/assets/7017516/24636210/83c89724-188d-11e7-8f96-960dd2cb4c24.png)

## After:
![screenshot_1491263316](https://cloud.githubusercontent.com/assets/7017516/24636209/83c785e6-188d-11e7-9185-c0904355480a.png)

## Test Plan

https://github.com/yihanseattle/ReactNativeAlertButtonText
This React Native app can reproduce the issue.

https://github.com/yihanseattle/TestAndroidAlertButtonText/tree/master
This is a native Android app that shows the difference when using two different AlertDialog.

ericvicenti commented:
What versions of Android have you tested this on? Can you post a screenshot of the "after"?

yihanseattle commented:
@ericvicenti it's on API 21. Tested on two real devices(Samsung S5 and Samsung Not 3) and an emulator. They are all on API 21. You are welcome to try the sample app on an emulator with API 21. The AlertDialog failed for me. Yes, will post a screenshot of "after".

yihanseattle commented:
Hi @ericvicenti, just added a screenshot of "after". and also added a native Android sample app that you can directly test the two AlertDialog. When I use the Support Library, the text is not cutting anymore. It only happened on a Samsung S5 with API 21. Then I verified that it happens on all devices with API 21. Let me know if you see the same.... thanks! :)
